src/app/pages/jails/jail-add/jail-add.component.ts

```diff
@@ -118,538 +127,644 @@ export class JailAddComponent implements OnInit {
- unrestricted access to all system addresses, <i>new</i> to\
- restrict addresses with <b>ip4_addr</b>, and <i>disable</i> to\
- stop the jail from using IPv4 entirely.'),
+      tooltip: T('Control the availability of IPv4 addresses. <br>\
+                 <b>Inherit</b>: Allow unrestricted access to all \
+                 system addresses. \
+                 <b>New</b> Restrict addresses with <b>ip4_addr</b>. \
+                 <b>Disable</b> Stop the jail from using IPv4 entirely.'),
       options: [{
-        label: 'New',
-        value: 'new',
-      }, {
         label: 'Inherit',
         value: 'inherit',
       }, {
-        label: 'None',
-        value: 'none',
+        label: 'New',
+        value: 'new',
+      }, {
+        label: 'Disable',
+        value: 'disable',
       }]
     },
     {
       type: 'checkbox',
       name: 'ip6_saddrsel',
       placeholder: T('ip6_saddrsel'),
-      tooltip: T('Check to disables IPv6 source address selection for the\
- prison in favor of the primary IPv6 address of the jail.'),
+      tooltip: T('Only availabled when the jail is not configured to \
+                 use VNET. Disables IPv6 source address selection for \
+                 the jail in favor of the primary IPv6 address of the \
+                 jail.'),
     },
     {
       type: 'select',
       name: 'ip6',
       placeholder: T('ip6'),
-      tooltip: T('This setting controls the availability of\
- IPv6 addresses. Possible values are <i>inherit</i> to allow\
- unrestricted access to all system addresses, <i>new</i> to\
- restrict addresses with <b>ip4_addr</b>, and <i>disable</i> to\
- stop the jail from using IPv6 entirely.'),
+      tooltip: T('Control the availability of IPv6 addresses. <br>\
+                 <b>Inherit</b>: Allow unrestricted access to all \
+                 system addresses. \
+                 <b>New</b> Restrict addresses with <b>ip6_addr</b>. \
+                 <b>Disable</b> Stop the jail from using IPv4 entirely.'),
       options: [{
-        label: 'New',
-        value: 'new',
-      }, {
         label: 'Inherit',
         value: 'inherit',
       }, {
-        label: 'None',
-        value: 'none',
+        label: 'New',
+        value: 'new',
+      }, {
+        label: 'Disable',
+        value: 'disable',
       }]
     },
     {
       type: 'input',
       name: 'resolver',
       placeholder: T('resolver'),
-      tooltip: T('Add lines to the jail <ins>resolv.conf</ins> file.\
- <b>Example:</b> <i>nameserver IP;search domain.local</i>. Fields must\
- be delimited with a semicolon (;), which are translated as new lines in\
- <ins>resolv.conf</ins>. Enter <i>none</i> to inherit the\
- <ins>resolv.conf</ins> file from the host.'),
+      tooltip: T('Add lines to the jail <b>resolv.conf</b>. \
+                 <b>Example:</b> <i>nameserver IP;search domain.local</i>. \
+                 Fields must be delimited with a semicolon (;), which \
```

Comment:
tense error: Fields must be delimited with a semicolon (;), which are translated > "is translated as a"

Reply:
fixed